geotrek/api/v2/serializers.py

```diff
@@ -360,6 +370,7 @@ class Meta:
 
     class TouristicModelSerializer(PDFSerializerMixin, DynamicFieldsMixin, serializers.ModelSerializer):
         geometry = geo_serializers.GeometryField(read_only=True, source="geom_transformed", precision=7)
+        accessibility = serializers.SerializerMethodField(read_only=True)
```

Comment:
```suggestion
        accessibility = serializers.SerializerMethodField()
```
stop to set read_only=True. SerializerMethodField has already this attribute to True. Then Viewsets are ReadOnly